### ISSUE TITLE
Focus mode

### ISSUE THREAD

**Issue by simon-ourmachinery** [OurMachinery/themachinery-public#327]
**Issue**
When entering the focus mode via menu there is visual indication that F11 will enter/exit the focus mode. This can be very confusing if one pays no attention in how they enter the mode.

**Possible Solutions**

- The chrome method 
When entering: After n time it will disappear
![image](https://user-images.githubusercontent.com/70658708/97185379-42ed8280-17a0-11eb-9dce-c27220fb5608.png)
When moving the mouse up:
![image](https://user-images.githubusercontent.com/70658708/97185421-4f71db00-17a0-11eb-8c1a-d040fd471931.png)

Advantage: e.g. in the simulation tab it feels like you are in the game.
Disadvantage: There will be something hiding the tab UI for a moment.

- Default menu:

> the default main menu bar, but when it's in focus mode there's also button to exit the focus mode in the upper right corner.
Advantage: Nothing will be over the UI
Disadvantage:   e.g. in the simulation tab it feels like you are not really in the game

- Using a tab bar:
![image](https://user-images.githubusercontent.com/70658708/97185807-cc04b980-17a0-11eb-9c68-9027680e3bf2.png)

Advantage: Nothing will be over the UI
Disadvantage:   

- e.g. in the simulation tab it feels like you are not really in the game
- Taught behaviour: The x on a tab closes the tab has different behaviour, it does not close the tab it just stops the focus mode.

**Comment by niklas-ourmachinery** [comment]
I think that for Simulate -- it is nice that it is truly fullscreen, because it ensures you get a resolution that matches that of the game etc. So the "Chrome" method would work best for that.

In other tabs, you might not really care about this... for example, in a regular editing tab, such as the scene tab or the graph tab. In those cases it might be better/less confusing to keep the regular menu bar and tab bar and just maximize the current tab.

I'm not sure if this means that we should have two different full screen modes... or if that's just confusing...

**Comment by simon-ourmachinery** [comment]
Not sure either... on the other hand the simulation tab is still something "special" because its the "heart" of what one is working on.

**Comment by niklas-ourmachinery** [comment]
True, but I think we want to avoid hard-coding special behaviors for specific tabs.
